lib/src/screens/buy/onramper_page.dart

```diff
@@ -6,25 +6,26 @@ import 'package:flutter_inappwebview/flutter_inappwebview.dart';
 import 'package:permission_handler/permission_handler.dart';
 
 class OnRamperPage extends BasePage {
-  OnRamperPage(this._onRamperBuyProvider);
+  OnRamperPage(this._onRamperBuyProvider, this.sellUri);
```

Comment:
we should rename the `onramperPage` to something generic since it's now used as the in-app web view page and extract its dependencies and just pass to it the URI and the title depending on the context

Reply:
done